cumulusci/tasks/marketing_cloud/get_user_info.py

```diff
@@ -0,0 +1,45 @@
+from logging import getLogger
+
+import requests
+
+from cumulusci.utils.http.requests_utils import safe_json_from_response
+
+from .base import BaseMarketingCloudTask
+from .mc_constants import MC_API_VERSION
+
+
+class GetUserInfoTask(BaseMarketingCloudTask):
+    """Retrieves user info of pertaining to the currently logged in user.
+    Sanitizes and returns the payload in self.return_values."""
+
+    logger = getLogger(__name__)
+
+    def _run_task(self):
+        endpoint = f"https://{self.mc_config.tssd}.auth.marketingcloudapis.com/{MC_API_VERSION}/userinfo"
+        headers = {
+            "Authorization": f"Bearer {self.mc_config.access_token}",
+            "Content-Type": "application/json",
+        }
+        try:
+            response = requests.get(endpoint, headers=headers)
+            response.raise_for_status()
+        except requests.exceptions.HTTPError:
+            self.logger.error(f"Exception occurred fetching user info: {response.text}")
+            raise
+
+        self.logger.info("Successfully fetched user info")
+        payload = safe_json_from_response(response)
+        self.return_values = self._sanitize_payload(payload)
```

Comment:
Should we also log the payload? The current output from running the task on the CLI is not very enlightening:

```
% cci task run marketing_cloud_get_user_info                                                                                                                                                                                 [01/19/22 16:30:42] Beginning task: GetUserInfoTask                                                                                                                                                                                                                              
                                                                                                                                                                                                                                                                                 
[01/19/22 16:30:43] Successfully fetched user info
```